lib/services.js

```diff
@@ -80,6 +82,20 @@ async function startHouseArrestService (udid, opts = {}) {
   return new HouseArrestService(socket);
 }
 
+async function startInstrumentService (udid, opts = {}) {
+  const osVersion = opts.osVersion || await getOSVersion(udid, opts.socket);
+  if (Number(osVersion.split('.')[0]) < 14) {
+    const socket = await startService(udid, INSTRUMENT_SERVICE_NAME, opts.socket);
+    // # after the ssl protocol is successfully handshake, I need to close the ssl protocol channel and use text transmission
+    // socket.ssl =  null
```

Comment:
for now it would be ok to throw an exception here that we don't support iOS below 14